views/default/admin/users/upload.php

```diff
@@ -13,8 +13,7 @@
  * @link http://arckinteractive.com/
  */
 elgg_load_css('upload_users.css');
-elgg_load_js('jquery.form');
```

Comment:
I didn't find any other references to jquery.form. Perhaps it has been left here accidentally?